test/Rx/Functional/Observable/ReturnObservableTest.php

```diff
@@ -5,25 +5,26 @@
 use Rx\Functional\FunctionalTestCase;
 use Rx\Observable\ReturnObservable;
 use Rx\Observer\CallbackObserver;
+use Rx\Scheduler;
 
 class ReturnObservableTest extends FunctionalTestCase
 {
     public function testReturnObservableSubscribeTwice()
     {
-        $o = new ReturnObservable("The Value");
+        $o = new ReturnObservable('The Value', Scheduler::getImmediate());
```

Comment:
That is the ReturnObservable test, so it's fine the way it is.
We do need to add some Observable::of tests in Operator